dev-docs/bidders/smartadserver.md

```diff
@@ -17,21 +18,21 @@ The Smart AdServer bidder adaptor requires setup and approval from the Smart AdS
+| `formatId` | required | The placement format ID                                                                                        | `1234`                                                                                                             | `integer` |
+| `domain`   | optional | The network domain (default see example)                                                                       | `'http://prg.smartadserver.com', 'https://prg.smartadserver.com'`                                                  | `string`  |
+| `target`   | optional | The keyword targeting                                                                                          | `'sport=tennis'`                                                                                                   | `string`  |
+| `currency` | optional | Override the default currency code (ISO 4217) of the ad request. (Default: `'USD'`)                            | `'EUR'`                                                                                                            | `string`  |
+| `bidfloor` | optional | Bid floor for this placement in USD or in the currency specified by the `currency` parameter. (Default: `0.0`) | `0.42`                                                                                                             | `float`   |
+| `appName`  | optional | Mobile application name                                                                                        | `'Smart AdServer Preview'`                                                                                         | `string`  |
+| `buId`     | optional | Mobile application bundle ID                                                                                   | `'com.smartadserver.android.dashboard'`                                                                            | `string`  |
+| `ckId`     | optional | Unique Smart AdServer user ID                                                                                  | `1234567890123456789`                                                                                              | `integer` |
+| `video`    | optional | Parameter object for instream video. See [video Object](#smartadserver-video-object)                           | `{}`                                                                                                               | `object`  |
+| `schain`   | optional | Supply Chain                                                                                                   | `'1.0,1!exchange1.com,1234,1,bid-request-1,publisher,publisher.com'`                                               | `string`  |
+| `userId`   | optional | The UserId Module                                                                                             | `{ britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId }` | `object`  |
```

Comment:
This list should just be up at the top as it's not actually bidder params that you ingest from the publisher, right?
```suggestion

```

Reply:
You are right, in my haste i forgot that it **isn't** a bidder parameter that we ingest, again good catch